app/models/movie_person_profile.rb

```diff
@@ -0,0 +1,59 @@
+class MoviePersonProfile
+  include ActiveModel::Validations
+
+  def initialize(person_id:, name:, bio:, birthday_and_age:, profile_path:)
+    @person_id = person_id
+    @profile_path = profile_path
+    @name = name
+    @bio = bio
+    @birthday_and_age = birthday_and_age
+  end
+
+  attr_accessor :person_id, :name, :bio, :birthday_and_age, :profile_path
+
+  validates :person_id, :name, :bio, :birthday_and_age, :profile_path, presence: true
+
+  WIKIPEDIA_CREDIT = {
+    starting: 'From Wikipedia, the free encyclopedia.',
+    trailing: 'Description above from Wikipedia.',
+    standard: 'Bio from Wikipedia.'
+  }.freeze
+
+  class << self
+    def parse_result(result)
```

Comment:
In case you haven't seen this yet (it's new to me!), using a section like:
```
class << self
  def something
  end
end
```
is a tidier way to organize a bunch of class-level methods like:
```
def self.something
end

def self.something_else
end
```
This way you don't have to have a bunch of loose `def self.something`s which could get accidentally mixed in with your other methods.